pkg/autoscaler/scaling/multiscaler.go

```diff
@@ -230,10 +235,7 @@ func (m *MultiScaler) Create(ctx context.Context, decider *Decider) (*Decider, e
 		}
 		m.scalers[key] = scaler
 	}
-	scaler.mux.RLock()
-	defer scaler.mux.RUnlock()
-	// scaler.decider is already a copy of the original, so just return it.
```

Comment:
The original code just returns `scaler.decider` instead of a copy. The new code returns a copy. But I think it is ok?

Reply:
I know, but
1) uniformity
2) return value is most likely unused outside tests
3) it might not be really true, since if we return the object it might be subsequently modified
So it's safer to return the copy as in other places.
Given it's not in the request path — it does not matter much to optimize this much here